docs/resources/samples/table-samples.md

```diff
@@ -83,6 +83,37 @@ function columnToSet(column: ExcelScript.TableColumn): string[] {
 }
 ```
 
+### Copy filtered contents to a new worksheet
+
+This sample filters the data in a table and then copies the filtered data to a new worksheet. It uses [`SpecialCellType`](/javascript/api/office-scripts/excelscript/excelscript.specialcelltype) to get the filtered cells in the original table and then uses `Range.copyFrom` to copy the data to the new worksheet.
+
+```TypeScript
+function main(workbook: ExcelScript.Workbook) {
+    // Get the table in the workbook named "StationTable".
+    const table = workbook.getTable("StationTable");
+
+    // Get the "Station" table column for the filter.
+    const stationColumn = table.getColumnByName("Station");
+
+    // Apply a filter to the table that will only show rows 
+    // with a value of "Station-1" in the "Station" column.
+    stationColumn.getFilter().applyValuesFilter(["Station-1"]);
+
+    // Get the filtered table data, as visible cells.
+    let headerRow = table.getHeaderRowRange();
+    let filteredTable = worksheet.getUsedRange();
+    let visibleRange = filteredTable.getSpecialCells(ExcelScript.SpecialCellType.visible); 
+
+    // Create a new worksheet for the filtered data. 
+    let newWorksheet = workbook.addWorksheet();
```

Comment:
Any reason we switch from `const` to `let` here?

Reply:
No, this was an oversight. Thanks for catching! Will fix.